bundles/org.corpus_tools.hexatomic.core/src/main/java/org/corpus_tools/hexatomic/core/undo/operations/AddLayerToGraphOperation.java

```diff
@@ -0,0 +1,64 @@
+/*-
+ * #%L
+ * org.corpus_tools.hexatomic.core
+ * %%
+ * Copyright (C) 2018 - 2020 Stephan Druskat, Thomas Krause
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package org.corpus_tools.hexatomic.core.undo.operations;
+
+import org.corpus_tools.hexatomic.core.SaltHelper;
+import org.corpus_tools.hexatomic.core.undo.ReversibleOperation;
+import org.corpus_tools.salt.graph.Graph;
+import org.corpus_tools.salt.graph.Layer;
+
+public class AddLayerToGraphOperation<L extends Layer<?, ?>> implements ReversibleOperation {
+
+  private final L layer;
+  private final Graph<?, ?, L> graph;
+
+  /**
+   * Create a undo operation for a layer that was added to a graph.
```

Comment:
```suggestion
   * A reversible operation for a layer that was added to a graph.
```

Using the verb IMHO is imprecise. Here and in the other classes implementing ReversibleOperation.